src/macros.rs

```diff
@@ -56,13 +58,15 @@ macro_rules! redis_command {
             return $crate::raw::Status::Err as c_int;
         }
 
+        let mandatory_acl_categories = AclCategory::from($mandatory_acl_categories);
         if let Some(RM_SetCommandACLCategories) = $crate::raw::RedisModule_SetCommandACLCategories {
-            let mut acl_categories = CString::new("").unwrap();
+            let mut acl_categories = CString::default();
             $(
-                if $mandatory_acl_categories != "" && $optional_acl_categories != "" {
-                    acl_categories = CString::new(format!("{} {}", $mandatory_acl_categories, $optional_acl_categories)).unwrap();
-                } else if $optional_acl_categories != "" {
-                    acl_categories = CString::new($optional_acl_categories).unwrap();
+                let optional_acl_categories = AclCategory::from($optional_acl_categories);
+                if mandatory_acl_categories != AclCategory::None && optional_acl_categories != AclCategory::None {
+                    acl_categories = CString::new(format!("{} {}", mandatory_acl_categories, optional_acl_categories)).unwrap();
+                } else if optional_acl_categories != AclCategory::None {
```

Comment:
I'd personally add a method for this `AclCategory::is_none()` and `is_some()`. It may clash with the interface of `Option` and confuse the reader, though, but there is no better alternative.